<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRcodeClass</class>
 <widget class="QWidget" name="QRcodeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRcode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame
{
	background: #1E1E20;
	border-radius: 10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_image">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_line" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#lineEdit
{
	background: rgba(234, 234, 234, 0.15);
	border-radius: 18px;
	font-size: 12px;
	font-family: 微软雅黑;
	color: #FFFFFF;
	padding-left:20px;
}</string>
           </property>
           <property name="placeholderText">
            <string>请输入内容</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_pushButton" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
border-radius: 14px;
border: 1px solid #01EEC3;
font-size: 12px;
font-family: 微软雅黑;
color: #01EEC3;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_qrcode">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>非库</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_dll">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>库</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_close">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QRcode.qrc"/>
 </resources>
 <connections/>
</ui>
